lars-reimann/api-editor
# feat: button to delete all annotations

Closes #165.

### Summary of Changes

As described in #165 under "Describe the solution you'd like".

### Screenshots (if necessary)

#### Button

![image](https://user-images.githubusercontent.com/2501322/140377510-0b341403-8567-4a94-ad13-22c72505b733.png)


#### Modal dialog

![image](https://user-images.githubusercontent.com/2501322/140377666-85b91dfe-3452-4cb7-9a25-6c396c65d7a0.png)


### Testing instructions

* Check that a modal dialog pops up before deletion
* Ensure that simply closing the modal does not affect the state
* Ensure that annotations in the Redux state (annotationSlice) and in the IndexedDB are gone after clicking "Delete"
